--- a/data/org.gnome.Dictionary.appdata.xml.in	2017-10-01 16:14:30.000000000 +0200
+++ b/data/org.gnome.Dictionary.appdata.xml.in	2019-09-13 12:59:27.476990558 +0200
@@ -28,7 +28,8 @@
     <kudo>AppMenu</kudo>
     <kudo>ModernToolkit</kudo>
   </kudos>
-  <project_group>GNOME</project_group>
+  <project_group>GNOME</project_group>  
+  <content_rating type="oars-1.1" />
   <translation type="gettext">gnome-dictionary</translation>
   <releases>
     <release version="3.26.0" date="2017-09-12">
